modules/47_regipol/regiCarbonPrice/declarations.gms

```diff
@@ -125,9 +125,9 @@ $endIf.cm_implicitFE
 
 $ifThen.co2priceSlope not "%cm_regipol_slope_beforeTarget%" == "off" 
 Parameter
-	p47_slope_beforeTarget(ttot,all_regi) "parameter to scale slope of co2 price trajectory in years before target year" / %cm_regipol_slope_beforeTarget% /
+	p47_slope_beforeTarget(ttot,ext_regi) "parameter to scale slope of co2 price trajectory in years before target year" / %cm_regipol_slope_beforeTarget% /
```

Comment:
the change is more generic than the first implementation allowing to create region targets for region groups (ex: EUR_regi) instead of only REMIND regions. as before